docker/README.md

```diff
@@ -191,6 +191,16 @@ whilst running the above `docker run` commands.
 ```
    --no-healthcheck
 ```
+
+## Disabling the healthcheck in docker-compose file
+
+If you wish to disable the healthcheck via docker-compose append your stanza with the following.
```

Comment:
Pardon my ignorance, but what's a stanza?

Reply:
From wiki; "In computing, a stanza consists of a related group of lines in a script or configuration file.[1] Formats depend on context"

So with docker-compose the entire code block for a container would be the stanza.

service:
  nginx:
    image: "nginx:latest"
    restart: unless-stopped
    healthcheck:
      disabled: true

Reply:
Right, thanks for clarifying. I feel like that's a bit of an obscure term, and that it would be better to replace it to something that is easier to understand (e.g. "append the following to your service configuration" or something similar), since the goal of this documentation is to be easily to understand by as many people as possible.

Reply:
no problem, replaced.

Reply:
Brilliant thanks :)